.github/workflows/ci.yml

```diff
@@ -36,63 +36,9 @@ jobs:
       with:
         go-version: ${{ secrets.GO_VERSION }}
     - run: GOPROXY=direct GOSUMDB=off GO111MODULE=on go get -u github.com/kisielk/errcheck; /home/runner/go/bin/errcheck -tags draft ./...
-#   static_check:
```

Comment:
I'm leery of removing these comments as the intention is uncomment at some point. These serve as a good reminder that they need to be gradually uncommented and included in the build. What do you think? If we have someone ready to take on issues that are found with these checks, we can begin uncommenting.

Reply:
@leecalcote we can restore it later. Also, anyone can again see the commented part by going through history. For now it can be removed imo.